pi-display/chase-kiosk-helper.py

```
@@ -0,0 +1,706 @@
+        psk = (form.get("psk", [""])[0] or "")
+        if not ssid:
+            PortalHandler.last_error = "Pick a network."
+            self._redirect("/")
+            return
+        PortalHandler.pending = {"ssid": ssid, "psk": psk}
+        _provision_event.set()
+        self._html(200, portal_connecting_page(ssid))
+
+
+def serve_portal() -> "socketserver.ThreadingTCPServer":
+    server = socketserver.ThreadingTCPServer(("0.0.0.0", PROVISION_HTTP_PORT), PortalHandler)
```

Ruff (0.15.13)

[error] Possible binding to all interfaces

(S104)